apps/actions/templates/a4_candy_actions/includes/reaction.html

```diff
@@ -0,0 +1,29 @@
+{% load i18n contrib_tags %}
+{% if action.type == 'comment' and action.verb == 'add' %}
```

Comment:
@phillimorland Comment on original PR:
so i did this because otherwise it was showing when i added a project when i had following at the end of the `if` inside the action__line and when I removed it, it had an empty space, not sure whats best to do?!
```
{% else %}
  {{ action }}
{% endif %}
```

Reply:
But I think we could now remove this as I am filtering for comments and 'add' anyway